extensions/ql-vscode/src/cli.ts

```diff
@@ -482,6 +482,30 @@ export class CodeQLCliServer implements Disposable {
     return await this.runJsonCodeQlCliCommand<QuerySetup>(['resolve', 'library-path'], subcommandArgs, 'Resolving library paths');
   }
 
+  /**
+   * Finds the language that a query is analyzing.
+   * @param queryUri The URI of the query
+   */
+
+  // Note: this currently fails for some queries, e.g. the "example" snippets..."
```

Comment:
Findings so far:
1. `findLanguage` works fine for all standard queries I've tried in the `<lang>/ql/src` folder
2. It fails for other queries, e.g. 
    - quick queries
    - example snippets in `<lang>/ql/examples/snippets`
    - [custom examples](https://github.com/github/vscode-codeql-starter/tree/main/codeql-custom-queries-cpp) in the starter workspace

For those failures, `codeql` seems unable to resolve the dependency in the qlpack 🤔 

<details><summary>Expand for log</summary>



```
Finding query language using CodeQL CLI: resolve queries -v --log-to-stderr --format bylanguage [redacted]\vscode-codeql-starter\ql\cpp\ql\examples\snippets\addressof.ql...
Sending shutdown request
Sent shutdown request
[2021-07-26 15:11:39] Calling plumbing command: codeql resolve library-path --qlconfig-file=[redacted]\AppData\Local\Programs\Microsoft VS Code\qlconfig.yml --query=[redacted]\vscode-codeql-starter\ql\cpp\ql\examples\snippets\addressof.ql --find-extractors --full-library-path=none --dbscheme=none --format=json
[2021-07-26 15:11:39] [DETAILS] resolve library-path> Resolving query at normalized path [redacted]\vscode-codeql-starter\ql\cpp\ql\examples\snippets\addressof.ql.
[2021-07-26 15:11:39] [DETAILS] resolve library-path> Found enclosing pack 'codeql-cpp-examples' at [redacted]\vscode-codeql-starter\ql\cpp\ql\examples.
[2021-07-26 15:11:39] [DETAILS] resolve library-path> Adding compilation cache at [redacted]\vscode-codeql-starter\ql\cpp\ql\examples\.cache.
[2021-07-26 15:11:39] [DETAILS] resolve library-path> Resolving library dependencies from [redacted]\vscode-codeql-starter\ql\cpp\ql\examples\qlpack.yml.
[2021-07-26 15:11:39] [ERROR] resolve library-path> ERROR: Dependency not found: codeql-cpp ([redacted]\vscode-codeql-starter\ql\cpp\ql\examples\qlpack.yml:1,1-1)
A fatal error occurred: Could not resolve library path for [redacted]\vscode-codeql-starter\ql\cpp\ql\examples
[2021-07-26 15:11:39] Exception caught at top level: Could not resolve library path for [redacted]\vscode-codeql-starter\ql\cpp\ql\examples
                      com.semmle.cli2.resolve.ResolveLibraryPathCommand.executeJSON(ResolveLibraryPathCommand.java:232)
                      com.semmle.cli2.resolve.ResolveLibraryPathCommand.executeJSON(ResolveLibraryPathCommand.java:45)
                      com.semmle.cli2.picocli.SimpleJsonSubcommand.executeInsistingOnJSON(SimpleJsonSubcommand.java:104)
                      com.semmle.cli2.picocli.PlumbingRunner.call(PlumbingRunner.java:54)
                      com.semmle.cli2.picocli.SubcommandCommon.callPlumbingInProcess(SubcommandCommon.java:123)
                      com.semmle.cli2.resolve.ResolveQueriesCommand.splitByLanguage(ResolveQueriesCommand.java:867)
                      com.semmle.cli2.resolve.ResolveQueriesCommand.printNonJsonResult(ResolveQueriesCommand.java:841)
                      com.semmle.cli2.resolve.ResolveQueriesCommand.printNonJsonResult(ResolveQueriesCommand.java:63)
                      com.semmle.cli2.picocli.SimpleJsonSubcommand.executeSubcommand(SimpleJsonSubcommand.java:84)
                      com.semmle.cli2.picocli.SubcommandCommon.executeWithParent(SubcommandCommon.java:432)
                      com.semmle.cli2.execute.CliServerCommand.lambda$executeSubcommand$0(CliServerCommand.java:67)
                      com.semmle.cli2.picocli.SubcommandMaker.runMain(SubcommandMaker.java:205)
                      com.semmle.cli2.execute.CliServerCommand.executeSubcommand(CliServerCommand.java:67)
                      com.semmle.cli2.picocli.SubcommandCommon.call(SubcommandCommon.java:448)
                      com.semmle.cli2.picocli.SubcommandMaker.runMain(SubcommandMaker.java:205)
                      com.semmle.cli2.picocli.SubcommandMaker.runMain(SubcommandMaker.java:214)
                      com.semmle.cli2.CodeQL.main(CodeQL.java:98)
```
</details>

Reply:
Woops, looks like I just needed to specify some `--additional-packs` 😄